usersession/client/client.go

```diff
@@ -333,8 +333,19 @@ func (client *Client) ServicesDaemonReload(ctx context.Context) error {
 	return err
 }
 
-func filterDisabledServices(all, disabled []string) []string {
+func filterDisabledServices(all []string, disabledSvcs map[int][]string, uid int) []string {
```

Comment:
does some doc comment need updating in this package?

Reply:
Good point! added